src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -704,7 +704,10 @@
 													<FontIcon
 														FontSize="12"
 														Foreground="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}"
-														Glyph="&#xE1CB;" />
+														Glyph="&#xE1CB;"
+														PointerEntered="TagIcon_PointerEntered"
+														PointerExited="TagIcon_PointerExited"
```

Comment:
Can we use visual states instead?

Reply:
Also, should the glyph change when hovering anywhere on the tag? I think it's a little difficult to discover if you have to hover over the icon for it to change.

Reply:
Almost done, there's still an issue with spacing. When you place the cursor over the space between the glpyh and the tag name `PointerExited` is raised.